spec/fixtures/files/alma/full_dump/three_records_one_bad_utf8.xml

```diff
@@ -0,0 +1,529 @@
+  <datafield tag="610" ind1="2" ind2="0">
+   <subfield code="a">Bolshoĭ teatr Rossii</subfield>
+   <subfield code="x">History.</subfield>
+  </datafield>
+  <datafield tag="650" ind1=" " ind2="0">
+   <subfield code="a">Conductors (Music)</subfield>
+   <subfield code="z">Russia (Federation)</subfield>
+   <subfield code="v">Biography.</subfield>
+  </datafield>
+  <datafield tag="650" ind1=" " ind2="0">
+   <subfield code="a">Conductors (Music)</subfield>
+   <subfield code="z">Soviet Union</subfield>
+   <subfield code="v">Biography.</subfield>
+  </datafield>
+  <datafield tag="650" ind1=" " ind2="0">
+   <subfield code="a">Musicians, Russian</subfield>
+   <subfield code="v">Biography.</subfield>
+  </datafield>
+  <datafield tag="910" ind1=" " ind2=" ">
+   <subfield code="c">F1301mon</subfield>
+  </datafield>
+  <datafield tag="980" ind1=" " ind2=" ">
+   <subfield code="a">20210811</subfield>
+   <subfield code="b">RuMoNKB</subfield>
+   <subfield code="f">NPRA1447</subfield>
+   <subfield code="i">39.00</subfield>
+   <subfield code="j">31.20</subfield>
+   <subfield code="n">11957701447</subfield>
+  </datafield>
+  <datafield tag="982" ind1=" " ind2=" ">
+   <subfield code="b">FY22</subfield>
+   <subfield code="c">rcppa</subfield>
+   <subfield code="n">2022</subfield>
+   <subfield code="q">32101108804400</subfield>
+  </datafield>
+  <datafield tag="986" ind1=" " ind2=" ">
+   <subfield code="h">ML422.V43</subfield>
+   <subfield code="i">C44 2021</subfield>
+  </datafield>
+  <datafield tag="950" ind1=" " ind2=" ">
+   <subfield code="c">2021-09-22 20:51:54 US/Eastern</subfield>
+   <subfield code="b">2021-09-22 20:51:56 US/Eastern</subfield>
+   <subfield code="a">false</subfield>
+  </datafield>
+  <datafield tag="852" ind1="0" ind2=" ">
+   <subfield code="b">recap</subfield>
+   <subfield code="c">pa</subfield>
+   <subfield code="h">ML422.V43</subfield>
+   <subfield code="i">C44 2021</subfield>
+   <subfield code="8">22901505800006421</subfield>
+  </datafield>
+  <datafield tag="952" ind1=" " ind2=" ">
+   <subfield code="d">2021-09-23 00:52:12</subfield>
+   <subfield code="8">22901505800006421</subfield>
+   <subfield code="a">2021-09-23 00:52:11</subfield>
+   <subfield code="b">ReCAP</subfield>
+   <subfield code="c">rcppa: RECAP</subfield>
+   <subfield code="e">false</subfield>
+  </datafield>
+  <datafield tag="876" ind1=" " ind2=" ">
+   <subfield code="0">22901505800006421</subfield>
+   <subfield code="a">23901505790006421</subfield>
+   <subfield code="j">0</subfield>
+   <subfield code="x">ACQ</subfield>
+   <subfield code="z">pa</subfield>
+   <subfield code="d">2021-09-23 00:52:11</subfield>
+   <subfield code="p">32101108804400</subfield>
+   <subfield code="y">recap</subfield>
+  </datafield>
+ </record>
+ <record>
+  <leader>01735nam a2200361 i 4500</leader>
+  <controlfield tag="005">20210421120000.0</controlfield>
+  <controlfield tag="008">210421s2021    ru ac    b    001 0сrus d</controlfield>
```

Comment:
The `c` in `0crus` is in fact an invalid UTF-8 character (it is represented as \x81 in Ruby) that causes Ruby to throw an exception when attempting to JSON.parse() a string that contains this value.